pkg/storage/ec/client.go

```diff
@@ -274,7 +274,11 @@ func (ec *ecClient) Repair(ctx context.Context, limits []*pb.AddressedOrderLimit
 }
 
 func (ec *ecClient) putPiece(ctx, parent context.Context, limit *pb.AddressedOrderLimit, data io.ReadCloser, expiration time.Time) (hash *pb.PieceHash, err error) {
-	defer mon.Task()(&ctx, "node: ", limit.GetLimit().StorageNodeId.String()[0:8])(&err)
+	nodeName := "nil"
```

Comment:
you can initialize to a zero value with `var nodeName string`

Reply:
By using `nil` we can search in the traces for empty traces a lot easier or count occurrences easier than if we just leave it blank.